src/overlays/actors/ovl_Eff_Dust/z_eff_dust.c

```diff
@@ -1,3 +1,10 @@
+/**
+ * @file z_eff_dust.c
+ * 
```

Comment:
Very minor but we usually have a slightly different style for the file comment
```suggestion
 * File: z_eff_dust.c
```

Reply:
According to [docs/Documenting.md](https://github.com/zeldaret/oot/blob/master/docs/Documenting.md#documenting-files), this should be documented as this. Maybe we should update `Documenting.md`.

Reply:
Either that or we will update all files at once if we choose to use the format described in the doc, but for now I think we should remain consistent with the other files